configuration/debian/tedge_agent/postinst

```diff
@@ -13,10 +13,10 @@ if ! getent passwd tedge-agent > /dev/null; then
 fi
 
 ### Create file in /etc/sudoers.d directory
-echo "%tedge-agent   ALL = (ALL) NOPASSWD: /etc/tedge/sm-plugins/apt" > /etc/sudoers.d/tedge-agent
+echo "%tedge-agent   ALL = (ALL) NOPASSWD: /etc/tedge/sm-plugins/[a-z]*" > /etc/sudoers.d/tedge-agent
 
 if [ -f "/etc/sudoers.d/010_pi-nopasswd" ]; then
-    echo "%tedge-agent   ALL = (ALL) NOPASSWD: /etc/tedge/sm-plugins/apt" > /etc/sudoers.d/tedge-agent-nopasswd
+    echo "%tedge-agent   ALL = (ALL) NOPASSWD: /etc/tedge/sm-plugins/[a-z]*" > /etc/sudoers.d/tedge-agent-nopasswd
```

Comment:
What if there are numbers, upper case letters, and special characters in the plugin name?

Reply:
Uppercase and numbers, definitely. Even if we decide to support special characters, please keep it minimal to some common chars in names like `-` and `_`.

Reply:
`[a-z]` limitation applies only the first character. Afterwards, user can use any character, this is `*` means. I agree `[a-zA-Z0-9]` but not convinced to add special characters.